finality-provider/cmd/fpd/daemon/daemon_commands.go

```diff
@@ -488,6 +488,63 @@ func runCommandEditFinalityDescription(cmd *cobra.Command, args []string) error
 	return nil
 }
 
+// CommandUnsafeDeleteMerkleProof removes merkle proof
+func CommandUnsafeDeleteMerkleProof() *cobra.Command {
+	var cmd = &cobra.Command{
```

Comment:
should we have a command to delete this? kkk seems a bit scary

Reply:
Yeah, maybe more docs about the consequences?